bridge/irc/irc.go

```diff
@@ -240,11 +241,16 @@ func (b *Birc) doSend() {
 	throttle := time.NewTicker(rate)
 	for msg := range b.Local {
 		<-throttle.C
+		username := msg.Username
+		if b.GetBool("Colornicks") {
+			checksum := crc32.ChecksumIEEE([]byte(msg.Username))
+			username = string(0x03) + string(checksum%0x10) + msg.Username + string(0x03)
```

Comment:
Doesn't work, but changing to
`username = fmt.Sprintf("\x03%d%s\x03", checksum%0x10, msg.Username)` works